app/models/invenio_rdm_record_converter.rb

```diff
@@ -10,7 +10,6 @@ class InvenioRdmRecordConverter < Sufia::Export::Converter
   SUBJECT_FIELDS = [:tag, :mesh, :lcsh, :subject_name, :subject_geographic]
   CIRCA_VALUES = ["ca.", "ca", "circa", "CA.", "CA", "CIRCA"]
   UNDATED = ["undated", "UNDATED"]
-  REFERENCE_FIELDS = ["bibliographic_citation", "part_of"]
```

Comment:
I think I had left this in thinking that we might need to use it again, but it's best to just remove it entirely.